### ISSUE TITLE
Developer runs Accessibility checks on web pages

### ISSUE THREAD

**Issue by erikamov** [cal-itp/cal-bc#205]
## User story / feature request

When a Developer runs tests
They can check Accessibility on web pages

### Acceptance Criteria

Locally:
1. Run tests on the project folder:`uv run manage.py test tests/cal_bc/system/test_project_lifecycle.py`
2. See no errors
3. Make a temporary change that would result in a accessibility violation
4. Run the same test (`uv run manage.py test tests/cal_bc/system/test_project_lifecycle.py`)
5. See tests failing with the message: `# Accessibility violations found.`
6. Fix the accessibility violation
7. Run the test again (`uv run manage.py test tests/cal_bc/system/test_project_lifecycle.py`)
8. See no errors

On PRs:
1. Create a PR
2. See actions running
3. Look for `Pytest / Run tests` (for example: https://github.com/cal-itp/cal-bc/actions/runs/30875825818/job/91886966454?pr=204)
4. Check for any accessibility violations
